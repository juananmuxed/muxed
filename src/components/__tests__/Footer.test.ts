import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';

import FooterVue from '../footer/Footer.vue';

describe('Footer', () => {
  it('Default', () => {
    const wrapper = mount(FooterVue);

    expect(wrapper.element).toMatchSnapshot();
  });
});
