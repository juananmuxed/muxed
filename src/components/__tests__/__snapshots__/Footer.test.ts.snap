// Vitest Snapshot v1

exports[`Footer > Default 1`] = `
<footer
  data-v-c343a562=""
>
  <span
    data-v-c343a562=""
  >
    <a
      data-v-c343a562=""
      href="https://github.com/juananmuxed"
      target="_blank"
    >
      GitHub
    </a>
  </span>
  <div
    class="m-badge primary"
    data-v-36b7fb34=""
    data-v-c343a562=""
    style="vertical-align: middle;"
  >
    <span
      class="label"
      data-v-36b7fb34=""
    >
      
      v2.6.0
      
    </span>
  </div>
  <span
    data-v-c343a562=""
  >
    © 2022 
    <a
      data-v-c343a562=""
      href="https://muxed.dev"
      target="_blank"
    >
      MuXeD
    </a>
  </span>
</footer>
`;

exports[`Footer > Props > Default 1`] = `
<footer
  data-v-c343a562=""
>
  <span
    data-v-c343a562=""
  >
    <a
      data-v-c343a562=""
      href="https://github.com/juananmuxed"
      target="_blank"
    >
      GitHub
    </a>
  </span>
  <div
    class="m-badge primary"
    data-v-36b7fb34=""
    data-v-c343a562=""
    style="vertical-align: middle;"
  >
    <span
      class="label"
      data-v-36b7fb34=""
    >
      
      v2.6.0
      
    </span>
  </div>
  <span
    data-v-c343a562=""
  >
    © 2022 
    <a
      data-v-c343a562=""
      href="https://muxed.dev"
      target="_blank"
    >
      MuXeD
    </a>
  </span>
</footer>
`;
